core/unix/module_private.h

```diff
@@ -36,6 +36,13 @@
 
 #include "configure.h"
 
+#ifndef ANDROID
+struct tlsdesc_t {
```

Comment:
Not understanding why this was moved: wouldn't being in a header cause more problems than in just the .c that uses it?

Reply:
`asm_offsets.cpp` needs to see the struct declaration.